addon/styles/partials/_uni-datepicker.scss

```diff
@@ -1,57 +1,54 @@
-$box-size: 42px;
+$cell-size: 40px;
+$cell-border-color: #eaebec;
+$day-color: #3d4a52;
+$disabled-background-color: #f3f3f4;
+$text-main-color: #161e22;
```

Comment:
These new colors should be on `uniq`, not here -- on the `color('calendar', ...)`